config.html

```diff
@@ -0,0 +1,657 @@
+<html>
+    <head>
```

Comment:
Would you mind switching to a 2 space indent for the HTML please?

Reply:
yeah, for sure. It'll be easier to read